src/js/jsx/shared/SplitButton.jsx

```diff
@@ -52,8 +52,7 @@ define(function (require, exports, module) {
             
             var buttonIcon;
             if (this.props.iconId) {
-                buttonIcon = (<SVGIcon viewBox="0 0 16 16" CSSID={this.props.iconId}
-                    iconPath={this.props.iconPath}/>);
```

Comment:
After switching to inline icons, it is no longer needed to refer the icon via its path. Another reason is that `iconPath` seems not in use any more.